doc/RecordingControl.xml

```diff
@@ -1470,6 +1470,62 @@ Change Request 2061, 2063, 2065, 2109</revremark>
       </variablelist>
       <para>The requested operation does not exist.</para>
     </section>
+    <section>
+      <title>OverrideSegmentDuration</title>
+      <para>OverrideSegmentDuration dynamically updates a RecordingConfiguration to produce smaller
+        recording segments for a limited time. This allows for a client to request faster access to recorded data while
+        reducing the overall storage costs during normal operation.
+      </para>
+      <para>
+        When the temporary override is active, the device should attempt to close its currently recording segment and then
```

Comment:
Is there any benefit for client/device to NOT close current recording segment ? Just clarifying as the  requirement is marked as recommended with "**should**" leaving a door open for implementations to keep current segment open?

Reply:
As clarified during telco, it's specified as should since it's pretty hard to enforce a behavior, since the device needs a new keyframe to close the segment which can introduce some delays. It's all best effort